plugins/notify.py

```diff
@@ -9,7 +9,7 @@ def send_notification(self, build_id, text):
         notification_list = self.load("notify_{}".format(build_id), '')
         self.say("{} BuildID: {}, Message: {}".format(notification_list, build_id, text)) 
 
-    @respond_to("^subscribe (@?)(?P<user>\w*) to (?P<build_id>\w*)")
+    @respond_to("^subscribe (@?)(?P<user>\w*) to (?P<build_id>\S+)")
```

Comment:
I would switch or whitespace with `\s` so we don't have to worry about whitespace as much.  Also, is it not possible to have `-`s in usernames?  Perhaps we should change the `\w` in the username to `\S`